### ISSUE TITLE
Unable to connect to the same room

### ISSUE THREAD

**Issue by REUZEE** [ianramzy/decentralized-video-chat#68]
I download the code and run the project locally followed the instructions. But one room could only accommodate one user. The user who accesses the same url will be seperated. And the log would print :
PS D: decentral ized-video-chat -master> npm start
video-chat201. 0.0 start D: dachuang decentralized- video-chat- master
node server. js
http://localhost:3000
darkfox: A client joined the room
darkfox: Broadcasting ready message
darkfox: Received token request
darkfox: Error: The requested resource 2010-04-01 Accounts AC/Tokens. json was not found

**Comment by Chaphasilor** [comment]
Do you have your STUN server (e.g. Twilio) set up correctly?

**Comment by REUZEE** [comment]
> Do you have your STUN server (e.g. Twilio) set up correctly?

I am sorry. I am not sure what's the stun server you said. I have signed up for free twilio account and got Account SID and Auth Token from the Twillio console and filled in credentials in the .env file. Then I successfully ran ngrok. Is it enough?

**Comment by Chaphasilor** [comment]
@REUZEE 

Alright, if you set up twilio inside the `.env` file your STUN server should be configured correctly.  
For local testing you don't nedd ngrok yet, try without it first in order to minimize error causes. Just run `npm start` and then navigate to `localhost:3000` and join a room, after you've done that copy the url of the room and open it in another tab in the same browser (beware of acoustic feedback).

If this doesn't work, I can investigate further :)

**Comment by agnelvishal** [comment]
Check this line https://github.com/ianramzy/decentralized-video-chat/blob/4b92b8a538a982bf65219c9cc120f41e0b9145c1/server.js#L81
Only 2 people can be present in a room as per that line

**Comment by Chaphasilor** [comment]
Exactly. However, OPs issue is that the second client can't join the room :)

**Comment by SaeedehAf** [comment]
Any update on this? I have the same issue. The second client can not join and it seems another room is being initiated. The second room also is waiting for someone to join.

**Comment by Chaphasilor** [comment]
@SaeedehAf Development of this project is halted for now. Please check the pinned issue.
